### ISSUE TITLE
Markets not returning anything

### ISSUE THREAD

**Issue by joevotto** [apseren/ap37#11]
The bitcoin market function doesn't return anything...has anything changed that needs to be updated in the API call?

**Comment by marcos10soares** [comment]
I'm having the same issue, and this works on desktop: 
```
function get(url, callback) {
    var xhr = new XMLHttpRequest();
    xhr.open('GET', url, true);
    xhr.onload = function () {
      if (xhr.status === 200) {
        callback(xhr.responseText)
      }
    };
    xhr.send();
  }
```
but still does not work on ap37, also trying to print xhr also prints nothing on ap37 so I'm not sure how to debug it.

**Comment by apseren** [comment]
Fixed by 398cae4
